src/util/webhooks.ts

```diff
@@ -87,3 +87,15 @@ export const emitWebhookFailedMetrics = (webhookName: string) => {
 	statsd.increment(metricWebhooks.webhookFailure, tags);
 
 }
+
+export const emitWebhookPayloadMetrics = (webhookName: string, size: number) => {
+	const tags = {
+		name: webhookName,
+		payloadSize: size.toString()
```

Comment:
We must never use values with unbounded variations as tags. It will explode the metric cardinality